build-tools/packages/build-cli/src/commands/vnext/check/latestVersions.ts

```diff
@@ -0,0 +1,97 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import type { ReleaseGroupName } from "@fluid-tools/build-infrastructure";
+import { isInternalVersionScheme } from "@fluid-tools/version-tools";
+import * as semver from "semver";
+import { releaseGroupArg, semverArg } from "../../../args.js";
+import { BaseCommandWithBuildProject, getVersionsFromTags } from "../../../library/index.js";
+
+type MajorVersion = number;
+
+export default class LatestVersionsCommand extends BaseCommandWithBuildProject<
+	typeof LatestVersionsCommand
+> {
+	static readonly summary =
+		"Determines if an input version matches a latest minor release version. Intended to be used in the Fluid Framework CI pipeline only.";
+
+	static readonly description =
+		"This command is used in CI to determine if a pipeline was triggered by a release branch with the latest minor version of a major version.";
+
+	static readonly args = {
+		version: semverArg({
+			required: true,
+			description:
+				"The version to check. When running in CI, this value corresponds to the pipeline trigger branch.",
+		}),
+		release_group: releaseGroupArg({ required: true }),
+	} as const;
+
+	public async run(): Promise<void> {
+		const { args } = this;
+		const buildProject = this.getBuildProject();
+		const releaseGroup = buildProject.releaseGroups.get(
+			args.release_group as ReleaseGroupName,
+		);
+
+		if (releaseGroup === undefined) {
+			this.error(`Package not found: ${args.release_group}`);
+		}
+
+		const versionInput = args.version;
+		const git = await buildProject.getGitRepository();
+		const versions = await getVersionsFromTags(releaseGroup, git);
+
+		if (!versions) {
+			this.error(`No versions found for ${releaseGroup.name}`);
+		}
+
+		// Filter out pre-release versions
+		const stableVersions = versions.filter((v) => {
+			return !isInternalVersionScheme(v);
+		});
```

Comment:
The comment doesn't seem to match the code. The code seems to be filtering out non-internal version scheme versions?